src/common.jl

```diff
@@ -1,13 +1,20 @@
 ## sample space/domain
 
 """
-`F <: VariateForm` specifies the form of the variate or
-dimension of a sample, univariate (scalar), multivariate (vector), matrix-variate (matrix).
+`F <: VariateForm` specifies the form of the variate or a sample.
 """
 abstract type VariateForm end
-struct Univariate    <: VariateForm end
-struct Multivariate  <: VariateForm end
-struct Matrixvariate <: VariateForm end
+
+"""
+`F <: NVariate{N}` specifies the form of the variate a sample for
```

Comment:
```suggestion
`F <: NVariate{N}` specifies the `size` of the variate a samples for
```

Reply:
How about "specifies the dimemsionality of the `size` ..."?